packages/lisk-transactions/src/utils/validation/schema.js

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+export const baseTransaction = {
+	$id: 'lisk/base-transaction',
+	type: 'object',
+	required: [
+		'id',
+		'type',
+		'amount',
+		'fee',
+		'senderPublicKey',
+		'recipientId',
+		'timestamp',
+		'asset',
+		'signature',
+	],
+	properties: {
+		id: {
+			type: 'string',
+			format: 'number',
+			minLength: 1,
+			maxLength: 20,
+		},
```

Comment:
Is it possible to set a range of values here?

Reply:
I think we need to add different validation function, ill look into it